lib/MusicBrainz/Server/Data/Recording.pm

```diff
@@ -427,7 +427,7 @@ sub find_recent_by_artists
         $search_url = sprintf('http://%s/ws/2/recording/?query=title:""',
                               DBDefs->SEARCH_SERVER);
     } else {
-        $search_url = sprintf('http://%s/recording/select?q=recording:*&wt=mbxml',
+        $search_url = sprintf('http://%s/recording/edismax?q=recording:*&wt=mbxml',
```

Comment:
Should this be edismax or dismax?

Reply:
This was only added to get the time that recordings were last indexed by the search server: 6cdbe908d137d7de9924f1aba2ebad33940631d1 (because recordings aren't indexed every three hours like the code previously expected). So I guess it doesn't matter what query it uses as long as there's a `created` field in the response.

Reply:
Of course, later we can delete this plus lib/MusicBrainz/Server/Controller/WS/js/LastUpdatedRecordings.pm and the code that uses it in root/static/scripts/release-editor/recordingAssociation.js once the new search is live, since it updates near-instantly.